.changeset/slimy-elephants-hear.md

```diff
@@ -0,0 +1,8 @@
+---
+'keystone-next-app': patch
+'@keystone-next/admin-ui': patch
+'@keystone-next/keystone': patch
+'@keystone-next/types': patch
+---
+
+Added graphql api route example for admin UI
```

Comment:
You don't need a changeset for `keystone-next-app` since it's not published but for the other packages, you should provide a more descriptive summary since seeing `Added graphql api route example for admin UI` in the changelog for any of the other packages wouldn't really make sense

Reply:
Done!